cmd/krel/cmd/changelog_data_test.go

```diff
@@ -724,7 +721,6 @@ we're pulling the correct go-runner arch from the manifest list.</p>
 <li>Service.beta.kubernetes.io/azure-load-balancer-disable-tcp-reset is removed.  All Standard load balancers will always enable tcp resets. (<a href="https://github.com/kubernetes/kubernetes/pull/94297">#94297</a>, <a href="https://github.com/MarcPow">@MarcPow</a>) [SIG Cloud Provider]</li>
 <li>Stop propagating SelfLink (deprecated in 1.16) in kube-apiserver (<a href="https://github.com/kubernetes/kubernetes/pull/94397">#94397</a>, <a href="https://github.com/wojtek-t">@wojtek-t</a>) [SIG API Machinery and Testing]</li>
 <li>Strip unnecessary security contexts on Windows (<a href="https://github.com/kubernetes/kubernetes/pull/93475">#93475</a>, <a href="https://github.com/ravisantoshgudimetla">@ravisantoshgudimetla</a>) [SIG Node, Testing and Windows]</li>
-<li>To ensure the code be strong,  add unit test for GetAddressAndDialer (<a href="https://github.com/kubernetes/kubernetes/pull/93180">#93180</a>, <a href="https://github.com/FreeZhang61">@FreeZhang61</a>) [SIG Node]</li>
```

Comment:
This is a commit with some code block which isn't release notes. Since the PR was marked with `release-notes-none`, I think `krel` should honor the label.